nixos/modules/virtualisation/incus.nix

```diff
@@ -11,78 +11,78 @@ let
-        coreutils
-        criu
-        dnsmasq
-        e2fsprogs
-        findutils
-        getent
-        gawk
-        gnugrep
-        gnused
-        gnutar
-        gptfdisk
-        gzip
-        iproute2
-        iptables
-        iw
-        kmod
-        libxfs
-        lvm2
-        lxcfs
-        minio
-        minio-client
-        nftables
-        qemu-utils
-        qemu_kvm
-        rsync
-        squashfs-tools-ng
-        squashfsTools
-        sshfs
-        swtpm
-        systemd
-        thin-provisioning-tools
-        util-linux
-        virtiofsd
-        xdelta
-        xz
-      ]
-      ++ lib.optionals (lib.versionAtLeast cfg.package.version "6.3.0") [
-        skopeo
-        umoci
-      ]
-      ++ lib.optionals (lib.versionAtLeast cfg.package.version "6.11.0") [
-        lego
-      ]
-      ++ lib.optionals config.security.apparmor.enable [
-        apparmor-bin-utils
+  path =
+    with pkgs;
+    [
+      "/run/wrappers/bin"
+      "${qemu_kvm}/libexec"
+      cfg.package
+
+      acl
+      attr
+      bash
+      btrfs-progs
+      cdrkit
+      coreutils
+      criu
+      dnsmasq
+      e2fsprogs
+      findutils
+      getent
+      gawk
+      gnugrep
+      gnused
+      gnutar
+      gptfdisk
+      gzip
+      iproute2
+      iptables
+      iw
+      kmod
+      libxfs
+      lvm2
+      lxcfs
+      minio
+      minio-client
+      nftables
+      qemu-utils
+      qemu_kvm
+      rsync
+      squashfs-tools-ng
+      squashfsTools
+      sshfs
+      swtpm
+      systemd
+      thin-provisioning-tools
+      util-linux
+      virtiofsd
+      xdelta
+      xz
+      zstd
+    ]
+    ++ lib.optionals (lib.versionAtLeast cfg.package.version "6.3.0") [
+      skopeo
+      umoci
+    ]
+    ++ lib.optionals (lib.versionAtLeast cfg.package.version "6.11.0") [
+      lego
+    ]
+    ++ lib.optionals config.security.apparmor.enable [
+      apparmor-bin-utils
 
-        (writeShellScriptBin "apparmor_parser" ''
-          exec '${apparmor-parser}/bin/apparmor_parser' -I '${apparmor-profiles}/etc/apparmor.d' "$@"
-        '')
-      ]
-      ++ lib.optionals config.services.ceph.client.enable [ ceph-client ]
-      ++ lib.optionals config.virtualisation.vswitch.enable [ config.virtualisation.vswitch.package ]
-      ++ lib.optionals config.boot.zfs.enabled [
-        config.boot.zfs.package
-        "${config.boot.zfs.package}/lib/udev"
-      ]
-      ++ lib.optionals nvidiaEnabled [
-        libnvidia-container
-      ]
-    )
-  }'';
+      (writeShellScriptBin "apparmor_parser" ''
+        exec '${apparmor-parser}/bin/apparmor_parser' -I '${apparmor-profiles}/etc/apparmor.d' "$@"
+      '')
+    ]
+    ++ lib.optionals config.services.ceph.client.enable [ ceph-client ]
+    ++ lib.optionals config.virtualisation.vswitch.enable [ config.virtualisation.vswitch.package ]
+    ++ lib.optionals config.boot.zfs.enabled [
+      config.boot.zfs.package
+      "${config.boot.zfs.package}/lib/udev"
```

Comment:
Doesn't that mean this would never have worked per https://github.com/NixOS/nixpkgs/pull/466666/files#r2577404366?

Reply:
Hmm, maybe not. But I've never noticed and am a zfs user.

Reply:
Fixed this with the `linkFarm` call.